### PR TITLE
Do not create Bot when existent bot authenticates

### PR DESCRIPTION
**Pull request by dpordomingo**

### CODE REVIEW

File: slack/client.go [hunk 1]
```diff
@@ -203,11 +204,16 @@ func (c *slackClient) AddBot(id, token string) {
 		}
 	}
 
+	if _, exists := c.bots[id]; exists {
```
Comment (mvader) [inline slack/client.go]: rename `exists` to `ok`

File: slack/client_test.go [hunk 2]
```diff
@@ -243,6 +243,21 @@ func TestSave(t *testing.T) {
...
+	storage := storage.NewMemory()
+	cli.SetStorage(storage)
+	cli.AddBot("1", "foo")
+	cli.AddBot("2", "foo")
+	cli.AddBot("1", "foo")
+
+	bots, _ := storage.LoadBots()
+	assert.Equal(2, len(bots))
+	assert.Equal(2, len(cli.loadedBots))
+
```
Comment (mvader) [inline slack/client_test.go]: remove empty line

File: slack/client.go (hunk 1 above)
Comment (dpordomingo) [inline slack/client.go]: done

File: slack/client_test.go (hunk 2 above)
Comment (dpordomingo) [inline slack/client_test.go]: done

**Pull request merged**